txprocessor/txprocessor.go

```diff
@@ -395,6 +395,10 @@ func (tp *TxProcessor) ProcessTxs(coordIdxs []common.Idx, l1usertxs, l1coordinat
 
 	// Process L2Txs
 	for i := 0; i < len(l2txs); i++ {
+		if l2txs[i].Type == common.TxTypeExit && l2txs[i].Amount.Cmp(big.NewInt(0)) == 0 {
```

Comment:
It seems to be OK.
However, is there a way to give any feedback to the user? Maybe block this in API level too?

Reply:
Marking as resolved as you already approved the PR that will do just do it